cf_xarray/accessor.py

```diff
@@ -897,7 +906,7 @@ def get_valid_axis(self) -> Set[str]:
 
         return set(varnames)
 
-    def get_valid_coords(self) -> Set[str]:
+    def coordinates(self) -> Set[str]:
```

Comment:
```suggestion
    @property
    def coordinates(self) -> Set[str]:
```

Let's add `@property` for all of these (except `keys`) so we can do `.cf.coordinates` instead of `.cf.coordinates()`

Reply:
Ok. I haven't done interactive PR stuff before but I will commit this and then add `@property` to the other new functions except `keys`.